modules/similarity-algorithms/pages/index.adoc

```diff
@@ -5,18 +5,55 @@ Similarity algorithms assign a score to a pair of vertices based on how similar
 This usually involves seeing if they have the same or similar neighborhoods.
 This section also includes distance algorithms, as distance is the inverse of similarity.
 
-The TigerGraph Graph Data Science Library provides the following similarity algorithms:
+The TigerGraph Graph Data Science Library provides the following:
 
-* xref:euclidean.adoc[]
+== Vector Similarity Functions
+
+[CAUTION]
+====
+Vector Similarity Functions are subcomponents for the Appropriate Nearest Nearest (ANN) algorithm which is still in beta.
+====
+
+[IMPORTANT]
+====
+Beta features are for personal or R&D use and not for production use.
+====
+
+=== Prerequisites
+
+
+.To use Vector Similarity Functions please install via one of the two methods below:
+* Via TigerGraph CLI run:
++
+[source, gsql]
+----
+tg box algos install <Approximate Nearest Neighbors Algorithm>
+----
+* Via GSQL terminal run:
++
+[source, gsql]
+----
+GSQL > BEGIN
+# Paste <Approximate Nearest Neighbors Algorithm> code after BEGIN command
+GSQL > END
+GSQL > INSTALL QUERY <Approximate Nearest Neighbors Algorithm>
+----
+
+For more information see our public GitHub repo https://github.com/tigergraph/gsql-graph-algorithms/blob/master/algorithms/Similarity/approximate_nearest_neighbors/README.md[Approximate Nearest Neighbors].
```

Comment:
```suggestion
```
The current UDFs are in the global ExprFunctions.hpp file for the library. They are not part of the ANN algorithm in particular. They are installed automatically.